samples/highcharts/blog/cognitive-impairment-no-labels/demo.js

```diff
@@ -0,0 +1,161 @@
+function padTo2Digits(num) {
+    return num.toString().padStart(2, '0');
+}
+
+function convertMsToHM(milliseconds) {
```

Comment:
Add a comment explaining why we need this function, so that implementers are not left with the false impression that they need to do all this just to print time correctly in Highcharts. In this case the X axis is a special case of time axis with a running number of hours.

Reply:
Good point